### PR TITLE
Update DBaaS backup documentation

### PR DESCRIPTION
**Pull request by oscarkraemer**

### CODE REVIEW

File: docs/cloud/dbaas/backups.md [hunk 1]
```diff
@@ -8,12 +8,12 @@
 All databases in DBaaS get automatically backed up about every 24 hours. Users can also manually
 backup their instances but that means that the next automatic backup will be taken about 24 hours
 after the last backup. All backups are stored for 90 days after which they get automatically
-deleted (not yet implemented). In the future, it will only be possible to delete backups automatically,
-users will not be able to manually delete their own backups.
+deleted, backups can not be manually delete by the user.
```
Comment (tolonenj) [inline docs/cloud/dbaas/backups.md:11]: ... can not be manually deleted by the user. (d's missing)

**Review by tolonenj — CHANGES_REQUESTED** [review]
I try this one (thanks Oscar for nominating me for these - good practice for me!) so My understanding is that you need to react to my comment? And it will launch a process or? Well, let's see.

File: docs/cloud/dbaas/backups.md (hunk 1 above)
Comment (oscarkraemer) [reply docs/cloud/dbaas/backups.md]: Fixed, I should not be allowed to use a keyboard.

**Comment by oscarkraemer** [comment]
I think once all conversations are resolved and somebody have approved the request the PR can be merged by somebody that is allowed to merge the changes.

**Review by tolonenj — APPROVED** [review]
Nice!

**Pull request merged**